src/ripple/shamap/Family.h

```diff
@@ -76,7 +76,7 @@ class Family
     isShardBacked() const = 0;
 
     virtual void
-    missingNode(std::uint32_t refNum) = 0;
+    missingNode(std::uint32_t refNum, uint256 const& nodeHash) = 0;
```

Comment:
This is the same signature as the method immediately below, but with the parameters reversed. Perhaps just call that method instead of changing this signature?

Reply:
That other function does other things, and the uint256 represents something else in the other function--namely, that one represents a ledger hash, and this one represents the hash of the object.

Reply:
Can we rename one or both `missingNode` overloads to indicate better what they do or are used for?  Different semantics for:
```c++
    virtual void missingNode(std::uint32_t, uint256 const&) = 0;
    virtual void missingNode(uint256 const&, std::uint32_t) = 0;
```

seems like an accident waiting to happen.

Reply:
fixed